Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -41,6 +41,9 @@ public override string VariableType
             get { return "Selection List"; }
         }
 
+        [IsSerializedForLocalRepository]
```

Comment:
try to set the attribute default value to "true" and confirm saving in XML only for "False"